content/developer/reference/user_interface/scss_inheritance.rst

```diff
@@ -0,0 +1,101 @@
+
+.. _contributing/scss_inheritance:
+
+================
+SCSS Inheritance
+================
+
+
+
+Overview
+========
+
+Managing SCSS assets in Odoo is not as straightforward as it is in some other environment, but it's highly efficient.
+
+Modularity is key, the inheritance scheme described further allows Odoo to:
+
+- customise Bootstrap CSS framework;
+- handle two different webclient designs (Community and Enterprise);
+- handle backend and frontend bundles separately (including the user's website design);
+- contextually load necessary assets only;
+- handle multiple color-schemes (eg. dark-mode);
+
+SCSS `!default` directive
+=========================
+
+"Direct variables’ overrides" are technically possible in SCSS but may lead to inconsistent results in complex environments like Odoo.
+
+.. code-block:: scss
+   :caption: :file:`library.scss`
+
+   $foo: red;
+
+.. code-block:: scss
+   :caption: :file:`customization_layer.scss`
+
+   $foo: blue; // -> Don't!
+
+Indeed, since the compilation process proceed across different interdependent bundles, re-assign a variable in the "wrong spot" may lead to unexpected cascade results.
```

Comment:
```suggestion
Indeed, since the compilation process acts across different interdependent bundles, re-assigning a variable in the "wrong spot" may lead to unexpected cascading results.
```